src/praisonai/praisonai/ui_agents/default_app.py

```
@@ -7,57 +7,62 @@
+                    "name": agent_def.get("name", "Agent"),
+                    "description": agent_def.get("description", "YAML-defined agent"),
+                    "instructions": agent_def.get(
+                        "instructions", agent_def.get("role", "")
+                    ),
+                    "model": agent_def.get(
+                        "model", os.getenv("PRAISONAI_MODEL", "gpt-4o-mini")
+                    ),
                     "icon": "🤖",
                 })
         return agents
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)